### PR TITLE
GSYE-502: Added validation for the configuration tree in the simulati…

### PR DESCRIPTION
**Pull request by spyrostz**
…on results.

## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=master
**GSY_FRAMEWORK_BRANCH**=master

### CODE REVIEW

File: src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py [hunk 1]
```diff
@@ -57,11 +57,17 @@ def __init__(self, is_scm: bool):
...
+        self.simulation_configuration_tree_validator = get_schema_validator(
+            "results_configuration_tree")
 
     def validate_simulation_raw_data(self, data: Dict):
         """Validate flattened_area_core_stats_dict."""
         self.simulation_raw_data_validator.validate(data=data, raise_exception=True)
 
+    def validate_configuration_tree(self, data: Dict):
+        """Validate configuration_tree dict."""
+        self.simulation_configuration_tree_validator.validate(data=data, raise_exception=False)
```
Comment (mrsaemir) [inline src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py:69]: is there any reason you decided not to raise exceptions here? We are at the moment not raising any exceptions in the `validate` implementation, so you can set `raise_exception=True` without any worries.

**Review by mrsaemir — COMMENTED** [review]
LGTM.
There's only one question to resolve. Thanks

File: src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py (hunk 1 above)
Comment (spyrostz) [reply src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py:69]: Good point, let me bring it back to True. Thanks!

**Review by mrsaemir — APPROVED** [review]
LGTM

**Pull request merged**